File: examples/kotlin/src/main/java/com/examples/classes/project_usage.kt
package com.examples.classes

import com.examples.classes.Project
import com.examples.classes.Task

fun createProject() {

    // [[project-usage]]
    // create a new project
    val title = "My Project"
    val prj = Project(title)
    assert(prj.title == title)
    // add tasks to it
    val task1 = Task("My Task")
    prj.addTask(task1)
    assert(prj.tasks()[0].getObjId() == task1.getObjId())
    // [[project-usage]]
}

File: examples/python/src/classes/project_usage.py
from project_pygen import Project
from task_pygen import Task


def create_project():
    # [[project-usage]]
    # create a new project
    title = 'My Project'
    prj = Project(title=title)
    assert prj.title == title
    # add tasks to it
    task1 = Task('First Task')
    prj.add_task(task1)
    task2 = Task('Second Task')
    prj.add_task(task2)
    assert prj.tasks() == [task1, task2]
    # [[project-usage]]
